### ISSUE TITLE
[Bug] remote_write is set statically in Prometheus's server_configmap.yaml

### ISSUE THREAD

**Issue by linhlam-kc** [kubecost/cost-analyzer-helm-chart#1784]
---
# For reproducible bugs, please provide the following

**Describe the bug**  
From an enterprise customer in the ticket https://kubecost.zendesk.com/tickets/2937 

```
Hey, I’ve run into an issue with 1.98.0-rc.4. We currently have a couple changes that we make to the scrape_configs. Since there wasn’t a way to tweak the jobs, we ended up with a full replacement of the prometheus.yml file in our values.yml. With 1.98.0, I see the additional feature for supporting AMP but the template appears to always add remote_write conditionally adding it like it did in 1.96.0. This causes us to get the error:
ts=2022-11-03T02:06:35.615Z caller=main.go:434 level=error msg=“Error loading config (--config.file=/etc/config/prometheus.yml)” err=“parsing YAML file /etc/config/prometheus.yml: yaml: unmarshal errors:\n  line 8: field remote_write already set in type config.plain”
```

In short, the `remote_write` is set statically in [server_configmap.yaml version: 1.98](https://github.com/kubecost/cost-analyzer-helm-chart/blob/releasebot/master-release-v1.98.0/cost-analyzer/charts/prometheus/templates/server-configmap.yaml#L17-L25)

In v1.96, `remote_write` is conditionally set instead: [server_configmap.yaml version: 1.96](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.96/cost-analyzer/charts/prometheus/templates/server-configmap.yaml#L17-L21)

**To Reproduce**  
N/A - the configuration is specific to customer's environment

**Expected behavior**  

`remote_write` need to be conditionally added as in the following example code snippet:

```
    global:
{{ $root.Values.server.global | toYaml | trimSuffix "\n" | indent 6 }}
{{- if $root.Values.global.amp.enabled }}
    remote_write:
    - url: {{ $root.Values.global.amp.remoteWriteService }}
      sigv4:
{{ $root.Values.global.amp.sigv4 | toYaml | indent 8 }}
{{- end }}
{{- if $root.Values.server.remoteWrite }}
    remote_write:
{{ $root.Values.server.remoteWrite | toYaml | indent 4 }}
```

**Screenshots**  
N/A

**Collect logs (please complete the following information):**
N/A

cc: @rossfisherkc @AjayTripathy

**Comment by dwbrown2** [comment]
@linhlam-kc good to close now that we merged #1784?!

**Comment by linhlam-kc** [comment]
@dwbrown2 Yes, it is good to close here. Thank you Webb.
